### PR TITLE
Add more documentation to new features

### PR DESCRIPTION
**Pull request by fmauch**
This PR adds more documentation for the changes currently introduced in the staging branch.

### CODE REVIEW

File: ur_robot_driver/doc/controllers.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+### `forward_joint_traj_controller`
+**Type:** [`pass_through_controllers/JointTrajectoryController`](http://wiki.ros.org/pass_through_controllers)
+
+This controller implements the
+[`control_msgs/FollowJointTrajectory`](http://docs.ros.org/en/latest/api/control_msgs/html/action/FollowJointTrajectory.html)
+interface. Instead of interpolating the trajectory on ROS side, this forwards the complete
+trajectory to the robot for execution leaving the robot in charge of calculating accelerations and
+speeds. Therefore, this will lead to motions most similar to what you could configure on the teach
+pendant.
```
Comment (gavanderhoorn) [inline ur_robot_driver/doc/controllers.md:152]: Should this say something about possible divergence from the commanded trajectory due to the OEM controller not sticking to the setpoints in that trajectory but "doing its own thing'?

Comment (fmauch) [reply ur_robot_driver/doc/controllers.md:152]: Thought about it when starting the paragraph, did forget it while writing the paragraph... Added in fb6c3ee

File: ur_robot_driver/doc/controllers.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+## Where are controllers defined?
+Controllers are defined in the `config/ur<model>_controllers.yml` files, e.g.
+[`ur10e_controllers.yml`](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/config/ur10e_controllers.yaml)
+for the UR10e robot.
+
+## How do controllers get loaded and started?
+As this driver uses ROS-control all controllers are managed by the
+[`controller_manager`](http://wiki.ros.org/controller_manager). During startup, a default set of
+running controllers is loaded and started, another set is loaded in stopped mode. Stopped
+controllers won't be usable, but their ROS interfaces are visible on the command line e.g. by
```
Comment (stefanscherzinger) [inline ur_robot_driver/doc/controllers.md:14]:
```suggestion
controllers won't be usable right away, but their ROS interfaces are still visible on the command line e.g. by
```

File: ur_robot_driver/doc/controllers.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+subset of all joints, but this won't be covered here.
+
+### `scaled_pos_joint_traj_controller` (Default motion controller)
+**Type:** [`position_controllers/ScaledJointTrajectoryController`](http://wiki.ros.org/scaled_joint_trajectory_controller)
+
+Similar to the non-scaled version it implements the
+[`control_msgs/FollowJointTrajectory`](http://docs.ros.org/en/latest/api/control_msgs/html/action/FollowJointTrajectory.html)
+interface. For controlling the robot it will send joint position commands.
+
+In contras to the non-scaled version, the speed slider position or any other speed-modifying action
```
Comment (stefanscherzinger) [inline ur_robot_driver/doc/controllers.md:81]:
```suggestion
In contrast to the non-scaled version, the speed slider position or any other speed-modifying action
```

File: ur_robot_driver/doc/controllers.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+[#119](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/issues/119).
+
+### `scaled_vel_joint_traj_controller`
+**Type:** [`velocity_controllers/ScaledJointTrajectoryController`](http://wiki.ros.org/scaled_joint_trajectory_controller)
+
+Similar to the non-scaled version it implements the
+[`control_msgs/FollowJointTrajectory`](http://docs.ros.org/en/latest/api/control_msgs/html/action/FollowJointTrajectory.html)
+interface. For controlling the robot it will send joint velocity commands.
+
+In contras to the non-scaled version, the speed slider position or any other speed-modifying action
```
Comment (stefanscherzinger) [inline ur_robot_driver/doc/controllers.md:138]:
```suggestion
In contrast to the non-scaled version, the speed slider position or any other speed-modifying action
```

**Review by stefanscherzinger — APPROVED** [review]
Some minor spell checking. Otherwise, great documentation!

**Pull request merged**